pom.xml

```diff
@@ -124,18 +124,18 @@
         <dependencies>
             <!-- Lightblue Core -->
             <dependency>
-                <groupId>com.redhat.lightblue.config</groupId>
-                <artifactId>core-config</artifactId>
+                <groupId>com.redhat.lightblue</groupId>
+                <artifactId>lightblue-core-config</artifactId>
                 <version>${lightblue-core.version}</version>
```

Comment:
Recommend moving away from properties for versions.  The release plugin will undo this and I think the version plugin has problems with it.  The release plugin will handle updating the hard coded versions just fine.